packages/transactions/lib/transactions.test.js

```diff
@@ -0,0 +1,9 @@
+// import { Transactions } from './transactions';
+
+describe('TransactionsModule', () => {
+  // let transactions = Transactions.getInstance();
+
+  it('Get instance', async () => {
+    expect(2).toBeDefined();
```

Comment:
I assume this a place-holder for a real test. 😄

Reply:
Yes, there are some tests on this file transactions-integration.test.js, which will test the integration with the core package.

Probably in this file we can try mock as much as possible